packages/admin-api-schema/lib/canary/index.js

```diff
@@ -0,0 +1,16 @@
+module.exports = [
```

Comment:
Was thinking of doing `fs.readDir()` approach which would read the actual contents of folder with schemas and definitions. Such approach would make the method `async` and would need more edge case handling to detect "schemas" only (we don't want to expose definitions as those are raw non-api specific JSON definitions). With this plain-list approach it enforces whoever is adding a new definition to expose it explicitly - we are than sure only "schema" not a "definition" are exposed.

Reply:
:+1: Explicit 'till it hurts - we can magic-ify this in the future if we need!